sources/multi/wtrlab.py

```diff
@@ -0,0 +1,133 @@
+# -*- coding: utf-8 -*-
+import logging
+import json
+import re
+
+import requests
+
+from lncrawl.core.crawler import Crawler
+from lncrawl.models import Volume, Chapter, SearchResult
+
+logger = logging.getLogger(__name__)
+
+
+class WtrLab(Crawler):
+    """
+        This site has multilingual novels, basically all MTL, supposedly through translators like Google
+        but the output seems pretty decent for that
+        The whole site obfuscates classes via Angular or some type of minifier
+        so the only constant HTML comes from display text & HTML IDs
+        But luckily all necessary data is stored in a consistent JSON that's always in the same script tag
+        Essentially the same framework as webfic though with some other keys, urls, etc.
+    """
+    base_url = ["https://wtr-lab.com/"]
+    has_manga = False
+    has_mtl = True
+
+    host = base_url[0][:-1]  # remove trailing /
```

Comment:
please assign this inside the `initialize` method

```
def initialize(self):
    self.host = self.base_url[0][:-1]
```

Reply:
Thanks for the input, moved that part into initialize() & used home_url instead, definitely nicer.
Also added a `return []` at the end of the search func so it returns an empty list if no results are found, looks cleaner that way.


f90208a